src/pages/markets/synthetic/_multipliers.js

```diff
@@ -130,7 +130,7 @@ const Multipliers = () => {
                 <Show.Eu>
                     <Text mt="1.6rem" color="grey-5" size="var(--text-size-xs)">
                         {localize(
-                            'Return to player (RTP) % for a multiplier of 200 and a 1 hour duration is about 95% on average',
+                            'Return to Player (RTP) for multiplier options using multiplier 100 and duration of 1 hour is in the range of 98-99 % on an average. Using a different multiplier or duration may affect the RTP.',
```

Comment:
im not sure but is this sentence already have translation for crowdin ?

Reply:
thats right, thanks for reminding, we will upload this onto crowdin later. as per discussion with PM, for EU related card now we can release first and then translate later 🙏